File: src/gui/objects/picker/MoodPicker.java
package gui.objects.picker;

import audio.tagging.Metadata;
import static main.App.APP;

/** Mood picker. */
public class MoodPicker extends Picker<String> {

	public MoodPicker() {
		super();
		itemSupply = () -> APP.db.getStringPool().getStrings(Metadata.Field.MOOD.name()).stream();
	}

}

File: src/player/main/sp/it/pl/gui/objects/picker/MoodPicker.kt
package sp.it.pl.gui.objects.picker

import sp.it.pl.audio.tagging.Metadata
import sp.it.pl.main.APP

/** Mood picker. */
class MoodPicker: Picker<String>() {
   init {
      itemSupply = { APP.db.itemUniqueValuesByField[Metadata.Field.MOOD].orEmpty().asSequence() }
   }
}